stdlib/array.ml

```diff
@@ -385,7 +385,7 @@ let sort cmp a =
 
 
 let cutoff = 5
-let stable_sort cmp a =
+let unsafe_stable_sort_segment cmp a ofs len =
```

Comment:
I would prefer loud names for those variables such as `init_ofs` and `init_len`, to avoid using them by mistake instead of one of the more local variables in the auxiliary functions.

Reply:
OK, will do.

(In fact, I have written a version of the code where the auxiliary functions are moved to the top level and commented; but that's a more invasive change.)